CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Main (unreleased)
 
 ### Features
 
+- Helm Chart: Add ingress and support for agent-receiver. (@ts-mini)
+
+- Helm chart: Add support for structuredConfig in values.yaml (@ts-mini)
```

Comment:
These should go into the changelog of the helm chart, not the root changelog: https://github.com/grafana/agent/blob/main/operations/helm/charts/grafana-agent/CHANGELOG.md

Reply:
fixed